openpype/pipeline/workfile/workfile_template_builder.py

```diff
@@ -1825,3 +1832,40 @@ def get_errors(self):
 
     def create_failed(self, creator_data):
         self._failed_created_publish_instances.append(creator_data)
+
+
+def build_first_workfile_from_template_builder():
+    print("#" * 100)
+    project_name = get_current_project_name()
+    project_settings = get_project_settings(project_name)
+
+    asset_name = get_current_asset_name()
+    asset = get_asset_by_name(project_name, asset_name)
+    task_name = get_current_task_name()
+    current_tasks = asset.get("data").get("tasks")
+    task_type = current_tasks[task_name]['type']
+    host = get_current_host_name()
+
+    build_workfile_profiles = project_settings[host]['templated_workfile_build']  # noqa
+
+    filtering_criteria = {
+        "task_names": task_name,
+        "task_types": task_type
```

Comment:
Is it correct that the key names are plural but the variables are singular?

Reply:
In the settings we can add multiple task names and task types for a single profile. But here we are passing the current task name and the current task type, so there will be only single value